package/Dockerfile

```diff
@@ -1,19 +1,26 @@
 FROM registry.suse.com/bci/golang:1.17 AS helm
 RUN zypper -n install git
-RUN git -C / clone --branch release-v3.8.0 --depth=1 https://github.com/rancher/helm
+RUN git -C / clone --branch v3.9.0 --depth=1 https://github.com/helm/helm
```

Comment:
> there is no differences between our fork and upstream

There are some changes: https://github.com/rancher/helm/commits/release-v3.8.0

For example: https://github.com/rancher/helm/commit/0576260935b059b9c95009b6f6cddc767da2bb8f
ForceAdopt is not in upstream: https://github.com/helm/helm/blob/main/cmd/helm/install.go

Reply:
I'll revert this, but I believe that we have some inconsistencies in this case, because Rancher is using multiple versions of Helm, including our fork, but also the upstream version directly.
- https://github.com/rancher/rancher/blob/release/v2.6/package/Dockerfile#L103
- https://github.com/rancher/rancher/blob/release/v2.6/package/Dockerfile#L120-L123

Reply:
That's true, it's because shell has a different usage than helm used in rancher. The goal is to use upstream helm, and I'm still confused why we ever forked helm v3 and why we couldn't work around it. That was a poor path to go down. But we can't revert without breaking some fleet use cases.